build.gradle

```diff
@@ -15,6 +15,7 @@ plugins {
 
 repositories {
     // Use Maven Central for resolving dependencies.
+    mavenLocal()
```

Comment:
This change allows Gradle to resolve dependencies from the local Maven repository (`mavenLocal()`), making it possible to use locally published versions of `iwf-java-sdk` during testing and development.